conans/client/command.py

```diff
@@ -499,12 +499,21 @@ def build(self, *args):
                             help="Optional. Local folder containing the conaninfo.txt and "
                                  "conanbuildinfo.txt files (from a previous conan install "
                                  "execution). Defaulted to --build-folder")
+        parser.add_argument("-c", "--configure", default=False, action="store_true",
+                            help="Execute the configuration step (variable should_configure=True)")
+        parser.add_argument("-b", "--build", default=False, action="store_true",
+                            help="Execute the build step (variable should_build=True)")
+        parser.add_argument("-i", "--install", default=False, action="store_true",
+                            help="Execute the install step (variable should_install=True)")
```

Comment:
`help="Execute the install step (variable should_install=True). When specified, configure/build step won't run unless --configure/--build specified"`